lib/services/notification.dart

```diff
@@ -21,43 +21,61 @@ class NotificationService {
 
     await _firestore
         .collection("/pushTokens")
-        .document(androidInfo.androidId)
-        .setData(data);
+        .doc(androidInfo.androidId)
+        .set(data);
   }
 
   Future<List<String>> getFCMTokens() async {
     List<String> list = [];
-    await _firestore.collection('/pushTokens').getDocuments().then((tokens) {
-      tokens.documents.forEach((token) {
-        list.add(token.data['deviceToken']);
+    await _firestore.collection('/pushTokens').get().then((tokens) {
+      tokens.docs.forEach((token) {
+        list.add(token.data.call()['deviceToken']);
       });
     });
     return list;
   }
 
   Future<Null> pushNotification(
-      {@required String title,
-      @required String msg,
-      @required String img,
-      @required String screen}) async {
+      {required String title,
+      required String msg,
+      required String img,
+      required String screen}) async {
     List<String> tokens = await getFCMTokens();
     final Map<String, dynamic> body = {
-      // "to": token,
+      // "to": 'fwGkbdNJRiSKRkOSQBMYi6:APA91bFG9P6YsOBaLJD8kEpUBd1VHn1W7SSrSehVdQ10X4hfQDxf32K2gUchpc1SYIwP78fxVOEecihsKXfxkdMDFtauaYLaFL_MHMhh4pmW36YYql_dskatPRhKuPFB6FCeD5IgQLwh',
       "registration_ids": tokens,
+      // ['fwGkbdNJRiSKRkOSQBMYi6:APA91bFG9P6YsOBaLJD8kEpUBd1VHn1W7SSrSehVdQ10X4hfQDxf32K2gUchpc1SYIwP78fxVOEecihsKXfxkdMDFtauaYLaFL_MHMhh4pmW36YYql_dskatPRhKuPFB6FCeD5IgQLwh', 'c9PwIj7jT6y7GxCyx7_6o7:APA91bGnW3uxeoaoqeFM7vxrWH2DtjXHve3f6QfKcERo7dJ5U4RMr3UNbZXZ96wbN11vpliiIQf6McSWKA-7ZiSg1utj3eE1Rf0lM8UrvR4AH9ULDEY_xySMG5bEwkaS1EzopThv8Ktm', 'dCydb-A8RDGwthLK2VR5JI:APA91bH0nF2NwSoGc8B849jIJZQmYhbuGWSrCyKfW81pvwuQIdmuKA2jKAB37eZSYp76uvk1VU2fUE2aiyKwhCRRUK0GEIXhLEpUeSgeO_nwQg5DDFt8ntuHJiijfSc0NUBQ34z2n5J3', 'e4FZT9cNSj6Z8WYAqZV3oP:APA91bHUlyYJ4I9gsXdA3yS1_eAKK_GylODWgUKWvXtRZ5XnYCAbaPoG6NmgGsMnyc9ylkvr-mY3E7VgBrXTC3QKIi0VObeIUBgYBTPi_C21lscS0CdkeTg7G_3njLqNcaq4HoQwbGjL', 'd4jwa-FJTFqiZdEa8S9996:APA91bEiezr7Kkh1XuSNWMNblaapBPDRe8M3gwbKN1vNIhb0kuHqy3kzggGmuygFF1h5ntYG4H1IkUGwSC5Yqwxs5F2wcRz95xnc_QtBETXwpFWGtMRRihgKHBREu-N-IK22et6IS010'],
```

Comment:
Why this?

Reply:
So the older device can still receive notifications will remove them with future updates

Reply:
Why are these tokens directly in the code file?
why not in the .env file

Reply:
Those are sample tokens that we used to test the functionality